aws/provider.go

```diff
@@ -242,6 +242,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_ssm_parameter":                    dataSourceAwsSsmParameter(),
 			"aws_subnet":                           dataSourceAwsSubnet(),
 			"aws_subnet_ids":                       dataSourceAwsSubnetIDs(),
+			"aws_vpc_ids":                          dataSourceAwsVpcIDs(),
```

Comment:
I'd recommend calling this data source `aws_vpcs` -- we may return more than IDs in the future. Unfortunately we have some more explicitly `_ids` named ones (like `aws_subnet_ids` above) that eventually may need to be deprecated should we need to return information other than IDs.